OpenOversight/tests/test_functional.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+
+def test_user_can_load_homepage_and_get_to_form(mockdata, browser):
+    browser.get("http://localhost:5000")
+
+    # Complainant loads homepage
+    assert "OpenOversight" in browser.title
+    browser.find_element_by_link_text("Try it").click()
+    time.sleep(5)
+    assert "Find an Officer" in browser.page_source
+
+
+def test_user_can_use_form_to_get_to_gallery(mockdata, browser):
+    browser.get("http://localhost:5000/find")
+
+    # Complainant submits default search params and gets list of officers
+    browser.find_element_by_name("submit-officer-search-form").click()
+    time.sleep(10)
+    assert "Digital Gallery" in browser.page_source
```

Comment:
Can we have better asserts here, like the results box is a div that exists with sufficient child elements? If not I'm okay with this for now.